### ISSUE TITLE
Question about the "Variable supply"

### ISSUE THREAD

**Issue by hbroer** [madMAx43v3r/mmx-node#118]
Hi,

I am not sure if I got it right. If netspace increases the rewards also increase, and if the netspace decreases the rewards also decrease? Or is this bound to the price on exchanges? I see a problem here because it is kind of unpredictable. It also can't decrease the supply. The supply will always be inflationary but never deflationary. Not sure how this should work into the opposite direction without (in worst case) set the rewards to 0 for a potential long time and therefor risk the networks security.

If it would have been this way with chia, then now not even the price would be low, also the rewards would be much lower. That would result in more people go away and sell what they got and then the rewards would even drop further. On the other hand there is no alternative to switch the plots to (not like with GPU mining where people can when ever they want switch to another network). So even if the price drops the supply stays high for a long time (compared to GPU mining) because the network slowly decreases (as seen with Chia). That pushes this delay of being profitable again further into the future.

This are just some thoughts. Looks like adding just this variable makes it very unpredictable. Even without this the market is very unpredictable and high risk for any investor. I don't think it will make the value more stable. It could cause the opposite and the network crash in worst case. The rewards are also not instantly sold by miners. Some sell later, some hodl, some sell directly. That's how the market tries itself to regulate the price. Algorythms did not do well to try to compensate that. But I have not much informations, like I said, just some thoughts.

Greetings
Hauke

**Comment by madMAx43v3r** [comment]
I don't think it even matters, none of your concerns would go away with fixed rewards.
Speculators and farmers will create a buffer on the market, so there is enough time for netspace to adapt slowly.

Really the only difference is, instead of creating most coins in the first year or two, they are created when demand for it arises. Without variable supply, it's like the money supply would be fixed. That would never work in a real economy, you'd get deflation if economy grows and no one would spend their money.

In my view, all fixed reward blockchains are crippled and ill suited to act as a "crypto currency".

**Comment by hbroer** [comment]
I understand what you try to archive. Would be an interesting experiment ;-) 

In a real world economy there are more factors. Resources are limited. No matter if you take a farm (there is just land of x m^2), or a oil pump (you can pump more in time but that only will make the field empty quicker). The income does not scale if a company only buys more tractors or pumps. The real economy also has limits to the amount of resources which are allowed to collect during a time period (fish for example). There are also mechanism to force the movement of values like taxes and (negative) interest charges. Ergo for example tries to implement this by automaticaly decrease the value of unspend coins after a time period.

IMO if a crypto currency should be a real currency and not just a ressource, it has to be kind of managed by lending with fees and sell and buy back to tie it loosly to other currencies and gold. The rewards should be limited to transaction fees and interest charges in this case. Real world currencies are "produced" out of nothing by loans via banks and that's very close to how currencies and their siblings, the promissory notes, work since a very long time (more than 2000 years). The system has to be completly different than any current crypto currency.

btw I realy would love to see a system which works like real currencies which rly could help some people. But I don't think that it can be archived completely decentralised. It may be too complex.

**Comment by madMAx43v3r** [comment]
Back in the day gold / silver coins were used as currency. In a way MMX works the same way, when silver coins are in short supply there's more incentive to mine for more silver. But also if there's an over-supply, you cant simply dump coins in the ocean.

The way fiat works is impossible to implement in a decentralized and permission-less way.

**Comment by hbroer** [comment]
> The way fiat works is impossible to implement in a decentralized and permission-less way.

that's the problem, but i don't think it is impossible. The solution just does not exist now.

**Comment by madMAx43v3r** [comment]
The issue is fiat lending is based on physical checks (ie. identity, job income, credit score, etc) which cannot be done on blockchain. Unless you do everything on-chain, address linked to identity, income paid on-chain, job contract on-chain, and outlaw any other financial system. Which might very well be in the plans, but not for MMX ;)

**Comment by hbroer** [comment]
yea I know. That's the problem I see too. IMO the only real problem is that you can't link one wallet to one human. Else you could do the calculations on the data of that wallet. And yea completely decentralised is complicated but I am not sure if that has to be the case. The central banks don't do that checks. The banks do, and in a perfect world they know their customers in person. At least 50 years ago before the system was that broken like it is now. :-D
